### PR TITLE
Media queries

### PR DESCRIPTION
**Pull request by rega1237**
# In this Branch Added
- Add the media queries for the desktop version and the mobile version respectively
- Add the style to make it look as close as possible to the design presented in Figma.
- Add all the images like social icons and background image
- Add the respective linters to check the code
- Update the README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -251,6 +264,8 @@ <h3 class="text-contact">
             </li>
           </ul>
         </form>
```
Comment (OLIPLICHE) [inline index.html:302]: - [ ] You have done an amazing job 👏 👏
- [ ] Please try again to stick as much as possible to the Figma design. 
- [ ] This picture below is from [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) 

fig1


- [ ] The Picture below is from your Design 

fig2

File: index.html [hunk 2]
```diff
@@ -278,7 +293,7 @@ <h3 class="text-contact">
         </ul>
         <p>rega1237@gmail.com</p>
         <div></div>
-      </section>
+      </footer>
     </main>
   </body>
```
Comment (OLIPLICHE) [inline index.html:334]: - [ ] From the section "project" in your code, Please make sure that the button `See Project` takes you to the GitHub repository. 

fact

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @rega1237, 

Good job so far 👏 👏👏
There are some issues that you still need to work on to get your project approved but you are almost there!


## Highlights
- Linter result is green ✔️
 - Commit messages are meaningful✔️
 - The pull request have a descriptive title telling you what changes are inside ✔️
 - The pull request have a short summary describing introduced changes in more detail ✔️
 - node_modules folder is in the .gitignore ✔️

## Required Changes ♻️

- [ ] **CHECK THE COMMENTS UNDER THE REVIEW**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]
Hi @rega1237 and @saifullah767 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**